core/src/main/kotlin/Kotlin/ExternalDocumentationLinkResolver.kt

```diff
@@ -46,6 +46,16 @@ class ExternalDocumentationLinkResolver @Inject constructor(
         connection.connectTimeout = timeout
         connection.readTimeout = timeout
 
+        // Add proxy basic authentication when proxyUser, proxyPassword is set and proxySet is true
+        val systemSettings = System.getProperties()
+        if (systemSettings.getValue("proxySet").toString() == "true"
```

Comment:
As far, as I debugged, Gradle already support basic authentification, so no need for such code in Gradle runner plugin. 
Isn't it better to use `java.net.Authenticator`? It looks like more straightforward approach.

Reply:
And why not to use `http.proxyUser` and so on?

Reply:
could you debug and verify also  Maven + proxy + basic authentication ?

I will try also  will try use "systemProp.jdk.http.auth.tunneling.disabledSchemes" too and let you know

we are no using Graddle at the moment so we need to fix Maven first